LibationWinForms/BookLiberation/ProcessorAutomationController.cs

```diff
@@ -305,7 +304,7 @@ public BackupLoop(LogMe logMe, IProcessable processable, AutomatedBackupsForm au
 		protected override async Task RunAsync()
 		{
 			// support for 'skip this time only' requires state. iterators provide this state for free. therefore: use foreach/iterator here
-			foreach (var libraryBook in Processable.GetValidLibraryBooks())
+			foreach (var libraryBook in Processable.GetValidLibraryBooks(ApplicationServices.DbContexts.GetContext().GetLibrary_Flat_NoTracking()))
```

Comment:
Curious about why you moved this here. It's certainly not right or wrong; I'm just curious about the choice.

Reply:
that DB call was previously inside IProcessibleExt.GetValidLibraryBooks. moving it into LibationWinForms allowed me to remove the ApplicationServices dependency from FileLiberator and make it DB ignorant.

Reply:
Ah, good call.